client/my-sites/email/email-providers-stacked-comparison/provider-cards/google-workspace-card.tsx

```diff
@@ -58,13 +59,10 @@ const getGoogleFeatures = (): TranslateResult[] => {
 	];
 };
 
-const googleWorkspaceCardInformation: ProviderCard = {
+const googleWorkspaceCardInformation: ProviderCardProps = {
 	className: 'google-workspace-card',
-	detailsExpanded: false,
 	expandButtonLabel: translate( 'Select' ),
-	onExpandedChange: noop,
 	providerKey: 'google',
-	showExpandButton: true,
```

Comment:
We don't need to define those default values since they will be overridden below.